docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -1,20 +1,28 @@
-# Adding, Updating and Deleting Users
+---
+title: Adding, Updating and Deleting Users
+summary: This article describes how to add, update and delete users in the Run:ai platform.
+authors:
+    - Jason Novich
+date: 2023-Dec-28
+---
 
 ## Introduction
 
-The Run:ai UI allows the creation of Run:ai Users. Users are assigned levels of access to all aspects of the UI including submitting jobs on the cluster.
+The Run:ai UI allows the creation of Run:ai **local** users. Local users are users that are only in the Run:ai platform. Users are assigned levels of access to all aspects of the UI including submitting jobs on the cluster.
 
 !!! Tip
     It is possible to connect the Run:ai UI to the organization's directory and use single sign-on (SSO). This allows you to set Run:ai roles for users and groups from the organizational directory. For further information see [single sign-on configuration](../runai-setup/authentication/sso.md).
 
 ## Working with Users
 
-You can create users, as well as update and delete users.
+You can create local users, as well as update and delete local users.
 
 ### Create a User
 
 !!! Note
-    To be able to review, add, update and delete users, you must have *System Administrator* access. To upgrade your access, contact a system administrator.
+
+    * To be able to review, add, update and delete users, you must have *System Administrator* access. To upgrade your access, contact a system administrator.
+    * When SSO is enabled, users created using the procedure below will be local users only. You can only provide SSO users with access to the system. You cannot create new SSO users.
```

Comment:
@jasonnovichRunAI in the last sentence maybe we should write "You shouldn't create new SSO users" instead of "You can't create new SSO users"?